mkt/websites/models.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from django.conf import settings
+from django.db import models
+
+import json_field
+
+from mkt.site.models import ModelBase
+from mkt.translations.fields import TranslatedField
+from mkt.tags.models import Tag
+from mkt.translations.fields import save_signal
+
+
+class Website(ModelBase):
+    default_locale = models.CharField(max_length=10,
+                                      default=settings.LANGUAGE_CODE)
+    url = TranslatedField()
```

Comment:
Regarding translating the URL:

**PROS:**
* Some Websites might need this

**CONS:**
* Make it harder to check for unicity (almost impossible to add a constraint for it).

By the way, URL checking is going to be a nightmare. People might submit the http version when we already have the https version, but also submit with extra query string parameters, extra subdomains, trailing slash or not, maybe different path... Do we want to restrict the number of websites par domain to one ? Probably not, but we have to do *something*.